StarWars-iOS/modules/home/controllers/HomeViewController.swift

```diff
@@ -12,17 +12,32 @@ class HomeViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Do any additional setup after loading the view.
-    }
-
-    /*
-    // MARK: - Navigation
+        // MARK: Regular way
```

Comment:
i see you have implemeted two ways to get the movies but do you want us to choose or why did you implemented the two ways?

Reply:
ok looks good